www/prisma/schema.prisma

```diff
@@ -8,36 +8,45 @@ datasource db {
 }
 
 model ForumPost {
-  id              Int      @id @default(autoincrement())
-  external_id     String   @unique
-  url             String   @unique
+  id              Int                @id @default(autoincrement())
+  externalId      String             @unique
+  url             String             @unique
   title           String
   username        String
   displayUsername String
-  category        String
-  about           String
-  firstPost       String
-  reaction        String
-  overview        String
-  tldr            String
-  createdAt       DateTime @default(now())
-  updatedAt       DateTime @updatedAt
+  categoryId      Int?
+  category        ForumPostCategory? @relation(fields: [categoryId], references: [id])
+  rawForumPostId  Int?
+  rawForumPost    RawForumPost?      @relation(fields: [rawForumPostId], references: [id])
+  about           String?
+  firstPost       String?
+  reaction        String?
+  overview        String?
+  tldr            String?
+  classification  String?
```

Comment:
`#could` move this to its own `ForumPostSummary` entity so that we don't throw away summaries we've generated